contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -805,6 +937,9 @@ def _fix_transitions(thing):
 
 
 def _simplify(thing):
+    if not thing:
+        return thing
```

Comment:
Is this meant for `None` or empty lists? Maybe a brief comment explaining the intent could be helpful.

Reply:
Both. If empty or None, there is nothing to simplify.